src/index.ts

```diff
@@ -343,7 +349,7 @@ export function mande(
     patch: _fetch.bind(null, 'PATCH'),
 
     // these two have no body
-    get: (url: string, options?: Options) => _fetch('GET', url, null, options),
+    get: (url?: string, options?: Options) => _fetch('GET', url, null, options),
```

Comment:
these will also need to get updated to `urlOrOptions?: string | number | Options`